lib/openhab/dsl/rules/triggers/cron.rb

```diff
@@ -72,22 +72,13 @@ def cron_expression_map
         # @param [Object] at TimeOfDay or String representing TimeOfDay in which to execute rule
         #
         #
-        def every(value, at: nil)
-          cron_expression = case value
+        def every(value, at: nil, attach: nil)
+          expression = case value
                             when Symbol then cron_from_symbol(value, at)
                             when Java::JavaTime::Duration then cron_from_duration(value, at)
                             else raise ArgumentExpression, 'Unknown interval'
                             end
-          cron(cron_expression)
-        end
-
-        #
-        # Create a OpenHAB Cron trigger
-        #
-        # @param [String] expression OpenHAB style cron expression
-        #
-        def cron(expression)
-          @triggers << Trigger.trigger(type: Trigger::CRON, config: { 'cronExpression' => expression })
+          append_trigger(Trigger::CRON, { 'cronExpression' => expression }, attach: attach)
```

Comment:
We still need the `cron` trigger:

```suggestion
          cron(expression, attach)
        end

        #
        # Create a OpenHAB Cron trigger
        #
        # @param [String] expression OpenHAB style cron expression
        #
        def cron(expression, attach: nil)
          append_trigger(Trigger::CRON, { 'cronExpression' => expression }, attach)
```

Reply:
Oops.. thanks.